lib/rubocop/cop/rails/active_record_callbacks_order.rb

```diff
@@ -44,9 +44,7 @@ class ActiveRecordCallbacksOrder < Cop
           after_touch
         ].freeze
 
-        CALLBACKS_ORDER_MAP = Hash[
-          CALLBACKS_IN_ORDER.map.with_index { |name, index| [name, index] }
-        ].freeze
+        CALLBACKS_ORDER_MAP = CALLBACKS_IN_ORDER.map.with_index.to_h.freeze
```

Comment:
This change isn't necessary to fix the problem, but I thought it was a nice simplification. 🙂

I'll happily take it out if it's not welcome, though.